[#7448 ][docs] Additional step for macOS Catalina to ensure "npm ci" does not fail (#7476)
yugabyte/yugabyte-db: docs/README.md

>
> Only now do `git` commands start to work normally. Now, too (but not before) `npm ci` works properly.

Note:

> For Catalina, an additional step maybe necessary to make sure xcode-select is pointing to the Xcode Developer directory using a command like this:

Reviewer: Probably don't need to mention Catalina here.

```suggestion
In some cases, an additional step may be necessary to make sure `xcode-select` is pointing to the correct Xcode or Command-Line Tools installation. For example:
```

> ```

Reviewer: ```suggestion
```sh
```

> xcode-select -s /Applications/Xcode.app/Contents/Developer

Reviewer: Per its manpage, `xcode-select --switch` should point just to the version of Xcode you want it to use, and should be run with `sudo`:

> Sets the active developer directory to the given path, for example **/Applications/Xcode-beta.app**. This command must be run with superuser permissions [...]

```suggestion
sudo xcode-select --switch /Applications/Xcode.app
```

Reviewer: And if someone installed the command-line tools standalone, they live in /Library/Developer/CommandLineTools...

I think we need to revisit this whole section with a fresh macOS install, and see what happens. You don't happen to have a fresh machine, do you? I don't, right now. :)

Author: I do not have a fresh machine at this point as well. Will review the other changes you suggested.
